lib/wsdl.js

```diff
@@ -1558,7 +1558,11 @@ WSDL.prototype.objectToXML = function(obj, name, namespace, xmlns, first, xmlnsA
       }
 
       if (!Array.isArray(child)) {
-        parts.push(['<', ns, name, attr, xmlnsAttrib, '>'].join(''));
+        var insNull = '';
+        if(typeof child === 'object' && child === null) {
```

Comment:
@herom Let me explain my reasoning for this extra check. SOAP API's are like grammar nazi's, I'm making absolutely sure here that this is in fact a NULL value, since you ONLY want the NULL values to have this kind of behavior. Please note `typeof null === 'object'`